core/templates/dev/head/css/oppia.css

```diff
@@ -2931,7 +2954,8 @@ div#ng-curtain {
   opacity: 0.7;
 }
 
-.oppia-rating-star-active {
+.oppia-rating-star-active,
+.protractor-test-rating-star:focus {
```

Comment:
Don't use .protractor-test-* classes in CSS -- those are only meant to be for e2e tests. Feel free to invent a new class name if you need to.